### PR TITLE
Add explanation about ViR (Vision Retention Network).

### PR DESCRIPTION
**Pull request by farrosalferro**
I have added explanations about ViR. However, it's only an overview so I don't include in-depth explanation about it. Ill explain it in more detail in the later chapter (probably using jupyter).

### CODE REVIEW

File: chapters/en/Unit 13 - Outlook/RMT.mdx [hunk 1]
```diff
@@ -34,6 +35,13 @@ This mechanism decomposes Self-Attention in images along horizontal and vertical
...
 ![MaSA vs Retention.png](https://huggingface.co/datasets/hf-vision/course-assets/blob/main/MaSA%20vs%20Retention.png)
 
+#### ViR
+![ViR.png](https://huggingface.co/datasets/hf-vision/course-assets/blob/main/ViR.png)
+
+Another work inspired by the RetNet architecture is the ViR, as discussed in the paper [ViR: Vision Retention Networks](http://arxiv.org/abs/2310.19731). In this architecture, the authors propose a general vision backbone with a redesigned retention mechanism. They demonstrate that ViR can scale favorably to larger image resolutions in terms of image throughput and memory consumption by leveraging the dual parallel and recurrent properties of the retentive network.
+
+The overall architecture of ViR is quite similar to that of ViT, except that it replaces the Multi-Head Attention with Multi-Head Retention (MHR). This MHR mechanism is free of any gating function and can be switched between parallel, recurrent, or chunkwise (a hybrid between parallel and recurrent) modes. Another difference in ViR is that the positional embedding is first added to the patch embedding, and then the [class] token is appended.
```
Comment (mmhamdy) [inline chapters/en/Unit 13 - Outlook/RMT.mdx]:
```suggestion
The overall architecture of ViR is quite similar to that of ViT, except that it replaces the Multi-Head Attention (MHA) with Multi-Head Retention (MHR). This MHR mechanism is free of any gating function and can be switched between parallel, recurrent, or chunkwise (a hybrid between parallel and recurrent) modes. Another difference in ViR is that the positional embedding is first added to the patch embedding, and then the [class] token is appended.
```

**Review by mmhamdy — COMMENTED** [review]
Hey @farrosalferro, thanks for adding the ViR section. This is really amazing🚀🚀. I just have one suggestion: I think we need to add a title and change the levels of the other headings accordingly. For an example title: `# Retention In Vision`, and then downgrade the other heading by one.

File: chapters/en/Unit 13 - Outlook/RMT.mdx (hunk 1 above)
Comment (farrosalferro) [reply chapters/en/Unit 13 - Outlook/RMT.mdx]: Thank you for the suggestions! I have fixed the document following your suggestions. Please take a look of it and let me know if there are something I need to fix. Thank you.

Comment (mmhamdy) [reply chapters/en/Unit 13 - Outlook/RMT.mdx]: Amazing! it's just that what I meant was to downgrade symbol-wise the rest of the headings (for example `###` -> `##` and `####` -> `###`). I think you did the opposite. Sorry for the confusion 😅

Comment (farrosalferro) [reply chapters/en/Unit 13 - Outlook/RMT.mdx]: I am sorry for the misunderstanding, I have fixed the document. Please check the changes and let me know if there is something I need to fix.

**Review by mmhamdy — APPROVED** [review]
Looking great🚀🚀, thanks @farrosalferro

**Pull request merged**